modules/gitops-release-notes-1-19-0.adoc

```diff
@@ -0,0 +1,158 @@
+
+Enhanced control over image pull policies for Argo CD components::
+The {gitops-title} Operator now provides administrators with flexible, multi-level control over the `imagePullPolicy` for container images used by the Argo CD, GitOpsService, and RolloutsManager components. This enhancement enables organizations to define a consistent image pulling strategy across all managed instances. The {gitops-title} Operator evaluates image pull policy settings in the following order:
+* Instance-level policy: Defined in the Custom Resource (CR) using the `.spec.imagePullPolicy` field. This overrides all other values for that instance.  
+* Subscription-level policy: Configured using the `IMAGE_PULL_POLICY` environment variable in the {gitops-title}  Subscription.  
+* Default policy: If no other policy is set, Argo CD defaults to `IfNotPresent`.
++
+The {gitops-title} Operator supports the following standard Kubernetes `imagePullPolicy` values:
+* `Always`: Argo CD always pulls the image from the registry.
+* `IfNotPresent`: Argo CD pulls the image only if it is not already present on the cluster node.
+* `Never`: Argo CD never pulls the image and uses only the locally available image.
++
+link:https://issues.redhat.com/browse/GITOPS-7721[GITOPS-7721]
+
+Delegated management of Argo CD notification settings::
+With this update, cluster administrators can delegate the management of Argo CD notification settings to application teams in additional namespaces. Application teams can configure their own triggers, templates, and subscriptions for their Applications without requiring access to the control plane namespace. Administrators enable this capability by adding the target namespace to the .`spec.sourceNamespaces` and `.spec.notifications.sourceNamespaces` fields in the Argo CD CR.
++
+link:https://issues.redhat.com/browse/GITOPS-7704[GITOPS-7704]
+
+Argo CD Image Updater available as a Technology Preview::
+With this update, Argo CD Image Updater is now available as a Technology Preview feature in {gitops-title}. Argo CD Image Updater automatically updates container images for Kubernetes workloads managed by Argo CD. It tracks image versions in applications as defined by ImageUpdater CRs and updates it by applying parameter overrides through the Argo CD API or by using the Git write-back method. The image updater supports applications built with Kustomize or Helm. In this release, the {gitops-title} Operator provides a productized container image of the Argo CD Image Updater and introduces a new property in the Argo CD CR to enable or disable the feature.
++
+The following example shows how to configure the Argo CD Image Updater feature in the Argo CD CR:
++
+[source,YAML]
+----
+spec:
+  sso:
+    provider: dex
+    dex:
+      volumeMounts:
+        - name: custom-cert
+          mountPath: /etc/dex/ssl
+          readOnly: true
+      volumes:
+        - name: custom-cert
+          secret:
+            secretName: dex-tls-secret
+----
```

Comment:
This section mentions `sso` instead of `imageUpdater`. Should be 
```
spec:
  imageUpdater:
    enabled: true
    env:
      - name: IMAGE_UPDATER_LOGLEVEL
        value: trace
    resources:
      limits:
        cpu: 500m
        memory: 1024Mi
      requests:
        cpu: 250m
        memory: 512Mi
```

Reply:
Fixed.